cves/kernel/CVE-2014-8173.yml

```diff
@@ -129,9 +131,9 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
+  code: false
+  code_answer: 
```

Comment:
Explain why was this did not involve unit tests, whether it was that there was no unit tests created for this file or the unit tests were not updated for the changes in the VCC.